Code/Engine/RendererCore/Decals/Implementation/DecalComponent.cpp

```diff
@@ -432,6 +432,19 @@ void ezDecalComponent::OnSimulationStarted()
     }
   }
 
+  // currently attaching a decal to a dynamic object does not make it appear on that object
```

Comment:
Should discuss whether / how to make it possible to attach decals to dynamic objects. Currently only works at runtime through surfaces and a special message that is sent.

Reply:
You can't really know here which dynamic object the decal should apply to. It can be the parent or the parent of the parent etc. I think we should just expose this "applyOnlyTo" object as property since we now have this object reference stuff in the editor.

---
In reply to: [400518330](https://github.com/ezEngine/ezEngine/pull/180#discussion_r400518330) [](ancestors = 400518330)